Blizzard.IO.RabbitMQ/RabbitPropertiesConverter.cs

```diff
@@ -12,23 +10,84 @@ public class RabbitPropertiesConverter :
     {
         public MessageProperties Convert(RabbitMessageProperties rabbitMessageProperties)
```

Comment:
Maybe we should just use the first method and assign to the present booleans true or false based on the null?

Reply:
Fixed